io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/internal/JsonInstanceConstants.java

```diff
@@ -15,12 +15,14 @@
 
 package eu.esdihumboldt.hale.io.json.internal;
 
+import eu.esdihumboldt.hale.io.json.JsonInstanceReader;
+
 /**
  * Constants used to generate JSON from instances.
  * 
  * @author Simon Templer
  */
-public interface InstanceJsonConstants {
+public interface JsonInstanceConstants {
```

Comment:
Any specific reason for renaming this class? It comes originally from the Instance to Json conversion and is still used there as well.